src/content/docs/cheerpj-applet-runner/03-getting-started/using-first-time.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Using the extension for the first time
+description: Tutorial for first-time users
+---
+
+This tutorial will guide you on the basic functionalities of the Applet Runner extension. We assume you have installed the extension on [Chrome](/cheerpj-applet-runner/getting-started/install-chrome) or [Edge](/cheerpj-applet-runner/getting-started/install-edge).
+
+<div class="mx-24">![](/cheerpj3/assets/applet-screenshots/homescreen.png)</div>
```

Comment:
The broken link error seems to be related to this image not being loaded/found. Could you check if the file exists in the repo or why this is happening?